client/callout.go

```diff
@@ -51,8 +51,13 @@ func (c *Callout) Get(url string, options ...RequestOption) ([]byte, error) {
 	}
 
 	httpClient := http.Client{}
-
-	req, err := http.NewRequest(http.MethodGet, url, nil)
+	var reqBodyReader io.Reader
+	if reqBody == "" {
+		reqBodyReader = nil
```

Comment:
We don't need this line since `var reqBodyReader io.Reader` sets `reqBodyReader` to `nil`